src/components/mdx-img/mdx-img.module.css

```diff
@@ -0,0 +1,28 @@
+.root {
+  background-color: var(--token-color-surface-primary);
+  box-shadow: var(--token-surface-base-box-shadow);
```

Comment:
Can we clarify with design if borders via box-shadow are meant to be on all images on DevDot? I feel like shouldn't be applying borders to all images and we haven't been in docs content. It doesn't seem like something we can automate because we'd have issues with some images such as window screenshots from Mac:

![image](https://user-images.githubusercontent.com/43934258/160627262-873ad108-3ecc-4ee4-af0d-3ef03e9b8be8.png)

Image location: https://developer.hashi-mktg.com/waypoint/docs/getting-started#viewing-the-web-ui

Reply:
☝️ This is a great question. I feel like this relates to the `hideBorder` prop in Learn... Maybe this is a reason we need to retain that prop.

In Docs contexts, we can set the default to `true` to match existing behaviour (at present I believe putting a border around images in Docs never happens?). Will confirm this with design - thanks for flagging, this'll definitely come up in the follow-up PR to use `Image` for Docs MDX content as well 👍

Reply:
Sounds fine to address in a follow-up PR. 👍🏻

I think the necessity of `hideBorder` depends on the necessity of having borders set on images. I have no context about why borders are being added to images, so I'm not currently convinced they're necessary for any content. My current stance is that all content needs to be aligned, and I think it's easier to bring alignment sooner rather than take it out for half our content later (like several PRs from now, not in a PR or two from now).

Reply:
I think I can live with this sort of style (dropshadow UI element inside a container with a border). It honestly looks very strange either way so at least we would be treating it consistently. The image with the border baked in is another story though, we might just need to find something that looks mostly good for all images and try to update older ones that do not work.

Reply:
@zchsh @kendallstrautman @braydenlove Okay so to summarize there are two separate issues here: border via `box-shadow` and rounded corners via `border-radius`.

I think we can apply consistent values across all images added via MDX and then come up with solutions (such as with the `noMargin` and `noBorder` props) for edge cases (such as images with built-in borders or drop shadows) as we go. Any objections?

Reply:
I chatted with @zchsh about it and we have a plan. I will have Zach walk you through what we discussed as it will be easier to follow :)

Reply:
☝️ @ashleemboyer yeah, this sounds good!

- Re: `box-shadow` for "border", this is coming from the design system. I think it's mostly just an annoying semantics thing, but is generally a non-issue - an alternate way of achieving a border-ish visual effect. Personally I'm going to continue to refer to the shadow on the `Image` component as a "border" cause that's what it looks like to me, even if it's not using the `border` CSS property. Happy to adjust this if anyone finds it confusing (I certainly do, but less confusing than talking about image's "box-shadow"s).
- Re: `border-radius`, with `<ImageConfig hideBorder />` authors can remove both the `box-shadow` and `border-radius`, giving them a very "plain" representation of their image. This will act as a fallback if we're not able to make all the content changes we want (content changes being: crop built-in borders out of images for Vault and Waypoint learn content before June 20th)

I'd propose we start by rolling out this change as-is. The issues we're seeing, being the black borders on some images now looking wonky due to duplicate bordering, were already technically "broken": they already had duplicate borders, the duplicate borders just didn't look quite as bad.

I don't think we need an additional props solution - if authors can't find a way to make an image work with a border around it, they can use `hideBorder` as an escape hatch.